ID: 14903
Title: Python client `jemalloc` error
Description:
### What happens?

When trying to do a basic query in Python:
```
#! /usr/bin/python3
import duckdb

duckdb.read_csv('20503_switch_dump.csv')
```
 it errors out with the following:
```
<jemalloc>: Out-of-range conf value: narenas:0
```

I am using the Python DuckDB client on Amazon Linux 2023.  The Python version information is as follows:
```
Python 3.9.16 (main, Jul  5 2024, 00:00:00) 
[GCC 11.4.1 20230605 (Red Hat 11.4.1-2)] on linux
```

DuckDB was installed through pip3. I tried both the stable and nightly version (`1.1.4.dev2005`).

The duckdb CLI client (downloaded directly from the website) does work.


### To Reproduce

- Create a fresh Amazon Linux 2023 EC2 t2.micro instance
- Install pip3: `sudo yum install python3-pip`
- Install Python DuckDB client: `pip install duckdb`
- Create test script `test.py`:
```
#! /usr/bin/python3
import duckdb

duckdb.read_csv('test.csv')
```
- Make script executable: `chmod +x test.py`
- Create test data `test.csv`:
```
a,b,c
1,2,3
4,5,6
7,8,9
```
- Running `./test.py` gives `<jemalloc>: Out-of-range conf value: narenas:0`

### OS:

Amazon Linux 2023

### DuckDB Version:

1.1.3

### DuckDB Client:

Python

### Hardware:

AWS t2.micro instance (Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz, 1GB memory)

### Full Name:

Stephen Daniel

### Affiliation:

REV Broadband

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a nightly build

### Did you include all relevant data sets for reproducing the issue?

Yes

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [x] Yes, I have